src/lib/assetBridger/erc20Bridger.ts

```diff
@@ -854,6 +854,65 @@ interface TokenAndGateway {
  * Admin functionality for the token bridge
  */
 export class AdminErc20Bridger extends Erc20Bridger {
+  private percentIncrease(num: BigNumber, increase: BigNumber): BigNumber {
+    return num.add(num.mul(increase).div(100))
+  }
+
+  public getApproveTokenRequestForCustomTokenRegistration(
```

Comment:
Why do we need this method as well as `getApproveGasTokenRequestForCustomTokenRegistration`?